network_importer/adapters/netbox_api/adapter.py

```diff
@@ -64,6 +63,39 @@ class NetBoxAPIAdapter(BaseAdapter):
 
     query_device_info_from_netbox = query_device_info_from_netbox
 
+    @staticmethod
+    def _is_tag_present(netbox_obj):
+        """Find if tag is present for a given object."""
+        if isinstance(netbox_obj, dict) and netbox_obj.get("tags", None):  # pylint: disable=no-else-return
+            return False
+        elif not isinstance(netbox_obj, dict):  # pylint: disable=no-else-return
+            try:
+                pass
+            except AttributeError:
+                return False
+        elif not netbox_obj["tags"]:
+            return False
+
+        for tag in config.SETTINGS.netbox.model_flag_tags:
+            if tag in netbox_obj["tags"]:
+                LOGGER.debug(
+                    "Tag (%s) found for object %s. Marked for diffsync flag assignment.", tag, netbox_obj,
+                )
+                return True
+        return False
+
+    @staticmethod
```

Comment:
would be good to convert this one to a classmethod to be able to call `cls. _is_tag_present` instead of `NetBoxAPIAdapter. _is_tag_present`, otherwise NetBoxAPIAdapter is hardcoded

Reply:
Confirmed with Damien that we can look to update this at a later date.